contracts/Lotto.sol

```diff
@@ -0,0 +1,287 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.7.3;
+pragma experimental ABIEncoderV2;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/utils/Counters.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "./ILottoNFT.sol";
+
+// TODO rename to Lottery when done
+contract Lotto is Ownable {
+    // Libraries 
+    // Counter for lottery IDs
+    using Counters for Counters.Counter;
+    Counters.Counter private lotteryIDCounter_;
+    // Safe math
+    using SafeMath for uint256;
+
+    // State variables 
+    // Instance of Cake token (collateral currency for lotto)
+    IERC20 internal cake_;
+    // Storing of the NFT
+    ILottoNFT internal nft_;
+
+    // Lottery size
+    uint8 internal sizeOfLottery_;
+    // Max range for numbers (starting at 0)
+    uint8 internal maxValidRange_;
```

Comment:
it can be public

Reply:
I'm assuming you want them both to be public. Will make it so, let me know if you want anything different.